apps/worker/src/app/workflow/usecases/send-message/send-message-sms.usecase.ts

```diff
@@ -48,11 +48,14 @@ export class SendMessageSms extends SendMessageBase {
     });
     if (!subscriber) throw new PlatformException('Subscriber not found');
 
+    const overrideSelectedIntegration = command.overrides?.sms?.integrationIdentifier;
+
     const integration = await this.getIntegration({
       organizationId: command.organizationId,
       environmentId: command.environmentId,
       channelType: ChannelTypeEnum.SMS,
       userId: command.userId,
+      identifier: overrideSelectedIntegration as string,
```

Comment:
same

Reply:
Because it is coming from the overrides, its type is unknown. If you have a trick for me to use, let me know🤩

Reply:
ohh, forgot that we have it not typed at all, I think that we have to improve it :) like it can be optional string there

Reply:
it is defined as an optional string in the command, but it is 'unknown' from the overrides